packages/auth/src/accessToken/service.ts

```diff
@@ -131,12 +131,15 @@ async function revokeByGrantId(
   deps: ServiceDependencies,
   grantId: string,
   trx?: TransactionOrKnex
-): Promise<number> {
-  return await AccessToken.query(trx || deps.knex)
+): Promise<Date> {
+  const revokedAt = new Date()
+  await AccessToken.query(trx || deps.knex)
     .patch({
       revokedAt: new Date()
     })
     .where('grantId', grantId)
+
+  return revokedAt
```

Comment:
Uses the `revokedAt` variable instead of storing another new `Date`.

```suggestion
): Promise<Date> {
  const revokedAt = new Date()
  await AccessToken.query(trx || deps.knex)
    .patch({
      revokedAt
    })
    .where('grantId', grantId)

  return revokedAt
```